draft-pauly-dprive-oblivious-doh.md

```diff
@@ -164,20 +175,16 @@ Clients MUST set the HTTP Content-Type header to "application/oblivious-dns-mess
 to indicate that this request is an Oblivious DoH query intended for proxying. Clients
 also SHOULD set this same value for the HTTP Accept header.
 
+A correctly encoded request has the HTTP Content-Type header "application/oblivious-dns-message",
+uses the HTTP POST method, and contains "targethost" and "targetpath" variables. The Proxy constructs 
```

Comment:
We should be a bit more pedantic about what `contains "targethost" and "targetpath" variables` means. URI templates don't necessarily translate to path variables. In other words, the requirement is not to contain variables, it's for the proxy to be able to extract them from the path. RFC 6570 seems to call this process "variable matching" so I'd say something along the lines of `If the proxy fails to match the "targethost" and "targetpath" variables from the path, it MUST treat the request as malformed.`

Reply:
Fixed.